Francisco/Marvel/src/script.js

```diff
@@ -0,0 +1,47 @@
+const apiKey = "9023e5031b526df7ec7abd1984db610e";
+const ts = "1";
+const hash = "bdaa8452ccbd7a0e73cb8fc0b7dabfb1";
+const url = `https://gateway.marvel.com:443/v1/public/characters?apikey=${apiKey}&ts=${ts}&hash=${hash}`;
```

Comment:
You should not store sensitive data on the PR. I think you should modify that.

Reply:
Alright!